### DISCUSSION TITLE
Come integrare strisce led bluetooth, bilancia sinocare e igrometro tramite ESPHOME

### DISCUSSION THREAD

**Discussion by emanueletocci** [indomus/forum#1329] [Hardware & Firmware]
Salve a tutti, oggi mi é arrivato un piccolo nodeMCU (ESP32) che ho configurato come gateway bluetooth/wifi (esphome) seguendo la guida indomus. Ora voglio collegarci alcuni dispositivi ma ho qualche problema con la creazione del firmware dal momento che é la prima volta che uso esphome (il firmware base giá l'ho "scritto").
In particolare vorrei integrare 2 strisce led bluetooth, una maxcio white (con dimmer caldo/freddo), l'altra con controller ledble RGB (magic home) e un piccolo igrometro ORIA (naturalmente sempre BT).  Volendo ( e se possibile) anche una bilancia sinocare. 
(Altre info sotto)
Devo procedere scrivendo manualmente il firmware o esiste qualcosa di "giá pronto" o adattabile? 
Grazie mille per l'aiuto.

P.S. Informazioni aggiuntive

Igrometro: **LINK RIMOSSO**

Striscia led Maxcio:**LINK RIMOSSO**

Striscia ledble: **LINK RIMOSSO**

**Comment by snakuzzo** [comment]
Sono vietati i link a store...rimuovo
Per quanto riguarda l'integrazione...devi cercare sul sito di esphome
